code/__HELPERS/roundend.dm

```diff
@@ -192,7 +192,7 @@
 	//Set news report and mode result
 	mode.set_round_result()
 
-	send2irc("Server", "Round just ended.")
+	send2irc("Server", "A round of [mode.name] just ended[mode_result == "undefined" ? "." : "with a [mode_result]."] Survival rate: [PERCENT(popcount[POPCOUNT_SURVIVORS]/GLOB.joined_player_list.len)]%")
```

Comment:
Turns out this can theorically cause a division by zero runtime.
a `var/survival_rate = GLOB.joined_player_list.len ? "[PERCENT(popcount[POPCOUNT_SURVIVORS]/GLOB.joined_player_list.len)]%" : ""` on the previous line may help.
Also you forgot to input a space between `"` and `with`.

Reply:
*scream